src/objective-c/GRPCClient/private/GRPCHost.m

```diff
@@ -107,6 +115,76 @@ - (grpc_call *)unmanagedCallWithPath:(NSString *)path completionQueue:(GRPCCompl
                                   gpr_inf_future(GPR_CLOCK_REALTIME), NULL);
 }
 
+
+- (BOOL)setSSLCredentials:(nullable NSString *)pemRootCerts
+           withPrivateKey:(nullable NSString *)pemPrivateKey
+            withCertChain:(nullable NSString *)pemCertChain
+                    error:(NSError **)errorPtr {
+  static NSData *kDefaultRootsASCII;
+  static NSError *kDefaultRootsError;
+  static dispatch_once_t loading;
+  dispatch_once(&loading, ^{
+    NSString *defaultPath = @"gRPCCertificates.bundle/roots"; // .pem
+    // Do not use NSBundle.mainBundle, as it's nil for tests of library projects.
+    NSBundle *bundle = [NSBundle bundleForClass:self.class];
+    NSString *path = [bundle pathForResource:defaultPath ofType:@"pem"];
+    NSError *error;
+    // Files in PEM format can have non-ASCII characters in their comments (e.g. for the name of the
+    // issuer). Load them as UTF8 and produce an ASCII equivalent.
+    NSString *contentInUTF8 = [NSString stringWithContentsOfFile:path
+                                                        encoding:NSUTF8StringEncoding
+                                                           error:&error];
```

Comment:
Actually, now that `GRPCHost` isn't getting a path anymore, but the contents of the file instead, do we really want the `errorPtr`? We're only using it for failing to load the default root certs, and that's not something we should be able to recover from.